engines/director/detection_tables.h

```diff
@@ -28,6 +28,7 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "director",			"Macromedia Director Game" },
 	{ "directortest",		"Macromedia Director Test Target" },
 	{ "directortest-all",	"Macromedia Director All Movies Test Target" },
+	{ "sgdirector",			"Macromedia Director紹介" }, // Macromedia Director Introduction
```

Comment:
We're going to forget what `sg` stands for. How about `directorshokai`?

Reply:
Good call, changed.